pyfranca/franca_lexer.py

```diff
@@ -118,22 +124,47 @@ def t_STRUCTURED_COMMENT(t):
         r"<\*\*(.|\n)*?\*\*>"
         t.lexer.lineno += t.value.count("\n")
 
-    # Identifier
+    # File name (quoted)
     # noinspection PyPep8Naming,PyIncorrectDocstring
     @staticmethod
-    def t_ID(t):
+    def t_FILE_NAME(t):
```

Comment:
It seems to me that `t_FILE_NAME` should be completely replaced by `t_STRING_VAL`. I can't think of a reason to keep it once constants are introduced.